libbeat/docs/security/api-keys.asciidoc

```diff
@@ -0,0 +1,116 @@
+[role="xpack"]
+[[beats-api-keys]]
+=== Grant access using API keys
+
+Instead of using usernames and passwords you can use API keys to grant
+access to {es} resources. API keys can be set to expire at a certain time
+and explicitly invalidated. Any user with the `manage_api_key` or `manage_own_api_key`
```

Comment:
```suggestion
and you can explicitly invalidate them. Any user with the `manage_api_key` or `manage_own_api_key`
```

Reply:
Suggested this edit because setting expiration and invalidating keys seem like two separate actions, so I've separated them grammatically.